### PR TITLE
Send the list of installed content objects with feedback

### PR DESCRIPTION
**Pull request by pazdera**
Related to: https://github.com/KanoComputing/kano-content/issues/21

cc @convolu @alex5imon

### CODE REVIEW

File: kano_feedback/DataSender.py [hunk 1]
```diff
@@ -357,6 +359,15 @@ def get_hdmi_info():
     return res + edid
 
 
+def get_co_list():
+    '''
```
Comment (convolu) [inline kano_feedback/DataSender.py]: It may be useful to wrap this in a try, except clause just in case the issue is being caused by the Content manager

Comment (pazdera) [inline kano_feedback/DataSender.py]: Good point, I'll do that tomorrow before merging

**Comment by pazdera** [comment]
@convolu Fixed your suggestion

**Comment by convolu** [comment]
@pazdera Thanks for taking it under consideration, :shipit: :+1:

**Comment by alex5imon** [comment]
Please merge :+1:

**Pull request merged**